_commands/basics/who.md

```diff
@@ -3,22 +3,65 @@
 
 who
 --
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
+## Description
```

Comment:
There needs to be a newline before the lines with `## ...`. Otherwise they don't render properly.